cms/djangoapps/contentstore/management/commands/backfill_orgs_and_org_courses.py

```diff
@@ -121,6 +182,55 @@ def confirm_changes(options, orgs, org_courseid_pairs):
     return answer.lower().startswith('y')
 
 
+def bulk_add_data(orgs, org_courseid_pairs, dry_run, activate):
+    """
+    Bulk-add the organizations and organization-course linkages.
+
+    Print out list of organizations and organization-course linkages,
+    one per line. We distinguish between records that are added by
+    being created vs. those that are being added by just reactivating an
+    existing record.
+
+    Arguments:
+        options (dict[str]): command-line arguments.
+        orgs (list[dict]): list of org data dictionaries to bulk-add.
+        org_courseid_pairs (list[tuple[dict, str]]):
```

Comment:
FWIW, you can also just add these type annotations to the function declaration itself now that we're firmly in Python 3. Then this space could be used to explain the details like what an org dict has in it, etc.

Reply:
Yeah, I've been nervous to add type annotations in the past, under the reasoning "what if I get them wrong, and then it's even harder to turn on MyPy?" But, if I get them wrong, I guess my code is probably broken anyway.

Reply:
Moved a bunch of types from the docstrings to annotations in a separate commit.